### ISSUE TITLE
Não salvou código

### ISSUE THREAD

**Issue by ThaisSantiago** [UNIVALI-LITE/Portugol-Studio#1002]
**Descrição do Problema**Salvei o código na aréa de trabalho, mas ele não apareceu lá, e acabei perdendo

**Mensagem de Erro (se houver)**:

**Captura de Tela / Arquivo.por**:

**Especificações do sistema**:

**Comment by AdsonEsteves** [comment]
@ThaisSantiago Isso acontece sempre?

**Comment by ThaisSantiago** [comment]
Não, mas aconteceu ontem de noite. Tinha salvo, mas logo após, já não tinha
mais registro.

Em ter., 7 de dez. de 2021 às 20:41, Adsu ***@***.***>
escreveu:
